<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMOFareListeFormu</class>
 <widget class="QDialog" name="TMOFareListeFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fare Liste Formu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Kaynaklar.qrc">
    <normaloff>:/resimler/fare.jpg</normaloff>
    <normalon>:/resimler/fare.jpg</normalon>:/resimler/fare.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">TMOFareListeFormu{

	background-image: url(:/resimler/fare.jpg);
}

QLabel {
   font:Bold; 
   color: #ffffff;
}
QGroupBox {
    font:Bold;
color: #ffffff;
}
QPushButton{font:Bold;}
QRadioButton{
    font:Bold;
color: #ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GPAramaBolumu">
     <property name="title">
      <string>FARE ARAMA BÖLÜMÜ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="GPParcaMarkasi">
        <property name="title">
         <string>Parça Markası</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="LPMarkasiDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPMarkasiDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPMarkasiFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPMarkasiFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PBAra">
        <property name="text">
         <string>ARA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="GPParcaModeli">
        <property name="title">
         <string>Parça Modeli</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="LPModeliDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPModeliDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPModeliFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPModeliFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPModeliIFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPModeliFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="PBKapat">
        <property name="text">
         <string>KAPAT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="TWTablo"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>RBPModeliFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1079</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>1145</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliIFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>911</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>871</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPModeliDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1089</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>1121</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPMarkasiDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PBKapat</sender>
   <signal>clicked()</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1222</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>1276</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PBAra</sender>
   <signal>clicked()</signal>
   <receiver>TMOFareListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1154</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>1277</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aramaYap()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
